app/src/main/java/com/twilio/voice/quickstart/IncomingCallNotificationService.java

```diff
@@ -0,0 +1,39 @@
+package com.twilio.voice.quickstart;
+
+import android.app.NotificationManager;
+import android.app.Service;
+import android.content.Context;
+import android.content.Intent;
+import android.os.IBinder;
+
+import com.twilio.voice.CallInvite;
+import com.twilio.voice.quickstart.VoiceActivity;
+import com.twilio.voice.quickstart.fcm.VoiceFirebaseMessagingService;
+
+public class IncomingCallNotificationService extends Service {
```

Comment:
I believe this should be a foreground service. Otherwise, users will be able to dismiss the notification. This is consistent with the Duo or the Android dialer behavior. Users are allowed only 4 actions:

1. Accept
2. Decline
3. Click the notification body (takes you to Activity)
4. Ignore the notification (the notification will eventually be cleared when the call is ended)